src/workloads/query/TenMDocCollection_IntId.yml

```diff
@@ -30,6 +30,17 @@ ActorTemplates:
         - OperationName: find
           OperationCommand:
             Filter: {_id: {^RandomInt: {min: 0, max: 9999999}}}
+- TemplateName: IDFieldPoint
+  Config:
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {_id: {^RandomInt: {min: 0, max: 9999999}}}
```

Comment:
Was this supposed to be no underscore?

Reply:
Oh okay one more interesting thing I noticed. If you run explain for this query it still does use the IDHACK query plan stage. But if you throw in an $eq like you do below - it disqualifies the IDHACK plan but it does then make you eligible for a 'PROJECTION_COVERED' plan, which will avoid fetching the whole document. [IDHACK will perform a fetch](https://github.com/10gen/mongo/blob/d0d74180a28b2c9eb9343dd0782f22ec81888eb0/src/mongo/db/exec/idhack.cpp#L123). So plausibly this query would get faster if it used $eq. Can we throw an $eq version in as well?

```js
winningPlan: {
      stage: 'PROJECTION_COVERED',
      transformBy: { _id: 1 },
      inputStage: {
        stage: 'IXSCAN',
        keyPattern: { _id: 1 },
        indexName: '_id_',
        isMultiKey: false,
        multiKeyPaths: { _id: [] },
        isUnique: true,
        isSparse: false,
```
vs: 
```js
  winningPlan: {
      stage: 'PROJECTION_SIMPLE',
      transformBy: { _id: 1 },
      inputStage: { stage: 'IDHACK' }
    },
```